### DISCUSSION TITLE
Introduce isSceneryNetwork json tag for track types

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#19710] [Feature requests]
I was thinking as an addition to #18392 to have a json tag that allows a track type to be tagged as a Scenery network, essentially a track type that's purely decorational, and will show up in the scenery menu rather than the ride menu.

I got the idea after I saw @spacek531 post a picture of a fence "track type" 
![image](https://user-images.githubusercontent.com/1516893/227326961-830b8404-e2f0-4fe3-9cfc-76649071fcd6.png)

This could also make it so the track hides with other scenery when hiding scenery, or doesnt have a ride ui.

Additionally, a cheat would be nice to have which allows you to tag any used track type as a "scenery network", so trackitecture will no longer show up in the ride overview list, or be moved to a new tab in that menu which lists scenery networks (which you can obviously not open or close). This cheat could probably be introduced already without the need for the track type objects.

**Comment by spacek531** [comment]
These kinds of CTRs usually abuse several game features, like using downward launch mode and closing the ride before it crashes. Many people use the vehicle editor plugin to change vehicle spacing.
